packages/utils/test/trimCanvas.tests.ts

```diff
@@ -15,7 +29,7 @@ describe('trimCanvas', () =>
 
         const trimmedImageData = trimCanvas(canvas);
 
-        expect(trimmedImageData.width).toEqual(9);
+        expect(trimmedImageData.width).toEqual(10);
```

Comment:
I believe this is a bug in the old implementation. Since it is drawn by `context.fillRect(10, 20, 10, 5)`, the width should be 10.

https://github.com/pixijs/pixijs/blob/e2f27bf276732172dac4c00fef54df5febec8ea4/packages/utils/src/media/trimCanvas.ts#L64-L71

Should be `bound.right < x + 1` here.

Anyway, with the new implementation in this PR, we can get the correct width.

Reply:
Yeah, good point!